curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-030.md

```diff
@@ -0,0 +1,134 @@
+---
+id: 60b69a66b6ddb80858c51594
+title: Part 30
+challengeType: 0
+dashedName: part-30
+---
+
+# --description--
+
+Move the dots into place by setting the `display` to `block`, the `margin` to `auto`, and the `margin-top` to `65%`.
+
+# --hints--
+
+Your `.black-dot` selector should have a `display` property set to `block`.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.black-dot')?.display === 'block');
+```
+
+Your `.black-dot` selector should have a `margin` property set to `auto`.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('.black-dot')?.margin.includes('auto'));
```

Comment:
Cannot read property `includes` of `undefined`!!!!
```suggestion
assert(new __helpers.CSSHelp(document).getStyle('.black-dot')?.margin?.includes('auto'));
```

Reply:
Y'know if I use jQuery this isn't a problem 🙂